src/tools/rust-analyzer/crates/ide-db/src/generated/lints.rs

```diff
@@ -1779,8 +1779,8 @@ The tracking issue for this feature is: [#110011]
 "##,
     },
     Lint {
-        label: "async_iter_from_iter",
-        description: r##"# `async_iter_from_iter`
+        label: "async_stream_from_iter",
```

Comment:
I don't believe we usually rename rust-analyzer's lint outputs. Please revert these.

Reply:
Oops, yes of course - done!